package.json

```diff
@@ -10,7 +10,8 @@
     "url": "git+https://github.com/DynamoDS/NotificationCenter.git"
   },
   "publishConfig": {
-    "access": "public"
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
```

Comment:
What is the reason to add this line?

Reply:
I added this to publish the npm package manually before I realized we got a pipeline. I'll remove it